Sources/MetadataHDWalletKit/Wallet/Coin.swift

```diff
@@ -33,6 +33,23 @@ public enum Coin {
         }
     }
 
+    // https://github.com/satoshilabs/slips/blob/master/slip-0132.md
+    public var publicKeyVersion: UInt32 {
+        switch self {
+        case .litecoin:
+            return 0x019da462
+        case .bitcoinCash: fallthrough
```

Comment:
`fallthrough`?  I thin adding a `fatalError("Not implemented")` and a TODO here would be better suited.

Reply:
`fallthough` to get the same value as `bitcoin`